modules/cloud_logs/main.tf

```diff
@@ -160,7 +160,8 @@ resource "random_string" "random_tenant_suffix" {
 
 locals {
   # If 'enable_platform_logs' is true, create a tenant in the same region as the ICL instance along with any region passed in the 'logs_routing_tenant_regions' list
-  logs_routing_tenant_regions = var.enable_platform_logs ? setunion([var.region], var.logs_routing_tenant_regions) : var.logs_routing_tenant_regions
+  logs_routing_tenant_regions     = var.enable_platform_logs ? setunion([var.region], var.logs_routing_tenant_regions) : var.logs_routing_tenant_regions
+  logs_routing_tenant_target_name = replace(substr("${local.instance_name}-l-r-target", 0, 32), "/[^a-zA-Z0-9]+$/", "")
```

Comment:
No need to affix `-l-r-target` to the name. We still want to try use the instance name. But trim it if its too long on contains invalid characters